src/modules/mc_pos_control/mc_pos_control_main.cpp

```diff
@@ -1440,6 +1497,36 @@ MulticopterPositionControl::task_main()
 						}
 					}
 
+					/* limit max change rate */
+					math::Vector<2> jerk_hor;
+					math::Vector<3> thrust_diff = thrust_sp - _thrust_sp_prev;
+
+					/* this is jerk times dt
+					 * we call it jerk so its clear what the intention is -
+					 * to limit the jerk, not the total acceleration.
+					 */
+					jerk_hor(0) = thrust_diff(0);
+					jerk_hor(1) = thrust_diff(1);
+
+					/* because we compare with jerk times dt, we multiply here as well */
+					float max_hor_jerk = (_params.acc_hor_max / ONE_G) * dt;
```

Comment:
@LorenzMeier If I understand the line above correctly then you are limiting the system to only increase acceleration by acc_hor_max / G in one second. So after one second the vehicle will have a velocity of only 0.5 m/s.
I think you should be removing the G in the denominator.